[microagent] Add pdflatex microagent (#7444)
All-Hands-AI/OpenHands: microagents/knowledge/pdflatex.md

---
name: pdflatex
type: knowledge
version: 1.0.0
agent: CodeActAgent
triggers:
- pdflatex
---

PdfLatex is a tool that converts Latex sources into PDF. This is specifically very important for researchers, as they use it to publish their findings. It could be installed very easily using Linux terminal, though this seems an annoying task on Windows. Installation commands are given below.

* Install the TexLive base

```
apt-get install texlive-latex-base
```

> * Also install the recommended and extra fonts to avoid running into the error [1], when trying to use pdflatex on latex files with more fonts.

Reviewer: ```suggestion
* Also install the recommended and extra fonts to avoid running into errors, when trying to use pdflatex on latex files with more fonts.
```

---

> name: latex

Author: maybe just pdflatex
```suggestion
name: pdflatex
```

> name: kubernetes

Reviewer 2: ```suggestion
name: latex
```

Author: ahh good catch!!!

type: knowledge
version: 1.0.0
agent: CodeActAgent
triggers:
- pdflatex

> - latex

Reviewer: ```suggestion
```

I'm not sure, but maybe latex is too generic for this tool?

Author: hmm yeah maybe true!
